src/current/v25.3/architecture/storage-layer.md

```diff
@@ -80,6 +80,43 @@ The SSTs within each level are guaranteed to be non-overlapping: for example, if
 
 <img src="{{ 'images/v21.2/lsm-with-ssts.png' | relative_url }}" alt="LSM tree with SST files" style="max-width:100%" />
 
+##### Write amplification
+
+_Write amplification_ measures the volume of data written to disk relative to the volume of data logically committed to the storage engine. When values are committed, CockroachDB writes them to the [write-ahead log (WAL)](#memtable-and-write-ahead-log) and then to [SSTables](#ssts) during flushes. [Compactions](#compaction) rewrite those SSTables multiple times over the value's lifetime. Most write amplification, and write bandwidth more broadly, originates from compactions.
+
+This is a necessary tradeoff, because if the storage engine performs too few compactions, the size of [L0](#lsm-levels) will get too large and an inverted LSM will result, which also has ill effects. In contrast, writes to the WAL are a small fraction of a [store]({% link {{ page.version.version }}/cockroach-start.md %}#store)'s overall write bandwidth and IOPs.
+
+Reducing write amplification helps lower disk bandwidth usage, CPU consumption for compactions, and storage costs.
```

Comment:
Perhaps I'm missing other context that's in this doc. But do we need to indicate some of the ways you can reduce write amplification? And it seems a bit tautological to say that reducing write amplification reduces CPU for compactions, when above we describe write amplification as being largely caused by compactions. Maybe the sentence could use a fresh approach overall? (I realize this sentence may have already existed, but it stands out to me more now that it's in its own section.)